src/GoTrueClient.ts

```diff
@@ -1229,13 +1242,37 @@ export default class GoTrueClient {
     if (options?.scopes) {
       urlParams.push(`scopes=${encodeURIComponent(options.scopes)}`)
     }
+    if (options?.isPKCE) {
+      const codeVerifier = this._generatePKCECodeVerifier()
+      setItemAsync(this.storage, this.storageKey, codeVerifier)
+      urlParams.push(`code_verifier=${encodeURIComponent(codeVerifier)}`)
```

Comment:
Instead of this array-based push logic, can you please use `URLSearchParams` as an object? (Also check if this exists in React Native?)

Reply:
Not sure if it works natively but should be fine [with a polyfill](https://justinnoel.dev/2020/12/08/react-native-urlsearchparams-error-not-implemented/)